docs/SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Currently, only the latest version of RMT is supported. Whenever we release a new version of RMT, we submit it to all supported versions of SLE and openSUSE which support RMT.
+
+## Reporting a Vulnerability
+
+**DO NOT CREATE AN ISSUE** to report a security problem. Instead, please contact the [SUSE Security Team](https://www.suse.com/support/security/contact/) instead.
```

Comment:
> Instead, ... instead.

😉

Reply:
Lol, if you don't mind, I made you a reviewer in the followup pr: https://github.com/SUSE/rmt/pull/609